backend/curriculum_tracking/tests/test_api_views.py

```diff
@@ -467,3 +472,142 @@ def test_team1_users_cannot_view_team2_burndown_snapshot_objects(self):
+        self.user_two_blue = core_factories.UserFactory(first_name='two_blue', is_superuser=False, is_staff=False)
+        self.user_one_red = core_factories.UserFactory(first_name='one_red', is_superuser=False, is_staff=False)
+        self.user_two_red = core_factories.UserFactory(first_name='two_red', is_superuser=False, is_staff=False)
+        self.super_user = core_factories.UserFactory(first_name='super_user', is_superuser=True)
+
+        self.blue_team.user_set.add(self.user_one_blue)
+        self.blue_team.user_set.add(self.user_two_blue)
+        self.red_team.user_set.add(self.user_one_red)
+        self.red_team.user_set.add(self.user_two_red)
+
+    def test_team_members_cannot_bulk_set_due_dates_for_the_team_they_belong_to(self):
+
+        project = RecruitProjectFactory(due_time=None)
+        card = AgileCardFactory(recruit_project=project)
+        card.reviewers.add(self.user_one_red)
+        card.assignees.add(self.user_two_red)
+        self.login(self.user_two_red)
+        url = f'{self.get_instance_url(pk=self.red_team.id)}bulk_set_due_dates/'
+        response = self.client.post(path=url, format="json", data={
+            'due_time': '2021-12-03T14:17', 'content_item': card.content_item.id,
+            'flavours': [1, 2, 3]
+        })
+        self.assertEqual(response.status_code, 404)
+
+    def test_team_members_cannot_bulk_set_due_dates_for_teams_they_dont_belong_to(self):
+
+        project = RecruitProjectFactory(due_time=None)
+        card = AgileCardFactory(recruit_project=project)
+        card.reviewers.add(self.user_one_blue)
+        card.assignees.add(self.user_one_red)
+        self.login(self.user_one_blue)
+        url = f'{self.get_instance_url(pk=self.red_team.id)}bulk_set_due_dates/'
+        response = self.client.post(path=url, format="json", data={
+            'due_time': '2021-12-03T14:17', 'content_item': '2', 'team': str(self.red_team.id),
+            'flavours': [1, 2, 3]
+        })
+        self.assertEqual(response.status_code, 404)
+
+    def test_super_user_can_bulk_set_due_dates_for_a_team(self):
+
+        project = RecruitProjectFactory(due_time=None)
+        card = AgileCardFactory(recruit_project=project, flavours=['JAVASCRIPT', 'PYTHON'])
+        card.reviewers.add(self.user_one_blue)
+        card.assignees.add(self.user_two_blue)
+        self.assertIsNone(card.due_time)
+        self.login(self.super_user)
+        url = f'{self.get_instance_url(pk=self.blue_team.id)}bulk_set_due_dates/'
+        response = self.client.post(path=url, format='json', data={
+            'due_time': '2021-12-03T14:17', 'content_item': card.content_item.id, 'team': str(self.blue_team.id),
+            'flavours': [card.flavours.first().id, card.flavours.last().id]
+        })
+
+        self.assertEqual(response.status_code, 200)
+        card.refresh_from_db()
+        self.assertTrue(card.due_time, '2021-12-03T14:17')
```

Comment:
This still has the `assertTrue()` which will pass regardless of what the second argument is. For example
```py
self.assertTrue(card.due_time, 'not even a date')
```